frontend/testing/playwright/tests/org-library/create-codelists.spec.ts

```diff
@@ -54,5 +54,24 @@ test('that it is possible to create a new codelist', async ({ page, testAppName
   await orgLibraryPage.codeLists.writeCodelistLabel(firstRow, firstRowLabel);
 
   await orgLibraryPage.codeLists.clickOnSaveCodelistButton();
-  await orgLibraryPage.codeLists.verifyThatCodeListIsVisible(codelistTitle);
+  await orgLibraryPage.codeLists.verifyThatCodeListIsVisible(CODELIST_TITLE);
+});
+
+test('that it is possible to search for and delete the new codelist', async ({
+  page,
+  testAppName,
+}) => {
+  const orgLibraryPage: OrgLibraryPage = await setupAndVerifyCodeListPage(page, testAppName);
+
+  await orgLibraryPage.codeLists.typeInSearchBox(CODELIST_TITLE);
+  await orgLibraryPage.codeLists.verifyThatCodeListIsVisible(CODELIST_TITLE);
+  await orgLibraryPage.codeLists.clickOnCodeListAccordion(CODELIST_TITLE);
+
+  const numberOfRowsInCodelist: number = 1;
+  await orgLibraryPage.codeLists.verifyNumberOfRowsInTheCodelist(numberOfRowsInCodelist);
```

Comment:
Typo in variable name
```suggestion
  const numberOfRowsInCodeList: number = 1;
  await orgLibraryPage.codeLists.verifyNumberOfRowsInTheCodelist(numberOfRowsInCodeList);
```